### DISCUSSION TITLE
Add product reviews to admin UI?

### DISCUSSION THREAD

**Discussion by coderreco** [swellstores/community#430] [Help] [answered]
Is there a way to show the reviews from `/products:reviews` somewhere in the admin UI? Would make handling approvals easier.

**Comment by ericingram** [comment, marked as answer]
We first introduced that model for users developing their own UIs and logic, but coincidentally we have been discussing whether to refine it recently. There are no plans to introduce a native UI, rather likely to lean on Apps to handle reviews. More to come on that topic.

**Reply by coderreco** [reply]
I assumed as much. Will likely explore my own UI for this in a future project.

Will there be a beta/testing channel for Apps that I could sign up for/apply for access to? I have a few ideas I would like to explore and would be interested in participating in that.

**Reply by ericingram** [reply]
We're in alpha stage right now with a small number of partners, but should reach open beta early next year. Look out for announcements on that :)

**Reply by ericingram** [reply]
Also note we'll make the an example app (reviews!) available on github before then, and there might be some interesting discussions here earlier we'd welcome you to participate in
